### PR TITLE
CEMS-2313 Migrate Partials

### PR DESCRIPTION
**Pull request by danmcadams**
Modify Render methods to use the new partial repository instead of gathering partials from the template repository. 

Create migration class to migrate old partials and images out of the template table and into the partial table.

### CODE REVIEW

File: src/DTS/AbstractEntity.php [hunk 1]
```diff
@@ -4,10 +4,10 @@
 namespace HomeCEU\DTS;
 
 
+use HomeCEU\DTS\Entity\GetPropertyTrait;
+
 abstract class AbstractEntity implements Entity {
-  public function get(string $property) {
-    return $this->{$property} ?? null;
-  }
+  use GetPropertyTrait;
```
Comment (ccopto) [inline src/DTS/AbstractEntity.php:10]: I hate traits

Comment (danmcadams) [reply src/DTS/AbstractEntity.php:10]: I don't mind them so much in tests (I actually just made a couple for some tests that shouldn't share a base class, but does need the same code multiple times.)

I couldn't think of a cleaner way to do this at the time.

**Comment by danmcadams** [comment]
@ccopto removed.

File: src/DTS/AbstractEntity.php (hunk 1 above)
Comment (ccopto) [reply src/DTS/AbstractEntity.php]: oh, lol, you didn't have to remove it... I don't like them, but they sometimes are useful... just don't abuse them

Comment (danmcadams) [reply src/DTS/AbstractEntity.php]: It was late when I added it. I had added a couple for testing and was just on a roll. I looked back through and found a better way to handle it.

**Pull request merged**